openshift/README.md

```diff
@@ -1,20 +1,31 @@
-h1. How to configure a CI/CD pipeline for ESM on OpenShift
+h1. How to quickly deploy to an OpenShift project
 
-- Create a project to house the Jenkins instance that will be responsible for promoting application images (via OpenShift ImageStreamTagS) across environment; the exact project name used was "esm".
-- Create the BuildConfiguration within this project using the ```oc``` command and "esm-build-template.json" file in the templates directory:
+- Request a set of Openshift projects (a typical set of projects is 'tools', 'dev', 'test', 'prod') for your new version of ESM on OpenShift from the CSI lab devops team via the Requests channel following this guide: https://pathfinder-faq-ocio-pathfinder-prod.pathfinder.gov.bc.ca/OCP/StartingANewProject.html
+- Your new 'tools' project will house the Jenkins instance that will be responsible for promoting application images (via OpenShift ImageStreamTagS) across the other three environments 'dev', 'test', and 'prod'. Your 'tools' project is typically set up in the naming convention of "[your-project-namespace]-tools" where [your-project-namespace] is auto-generated by the CSI lab devops team.  In the existing esm-server project, the tools name used was "esm".  This was set using an old naming convention and one of the peculiarities of OpenShift is that once a project is named it cannot be renamed, which is why the lab now assigns random short strings as project name prefixes.
+- In the ./templates/params directory, copy the Virtual Environment folder called 'VENV' into a new folder with the initials of your new project in the params directly.
```

Comment:
File paths are confusing. This one for example should be ./openshift/templates/params